e2e-tests/src/page-objects/theia-single-input-dialog.ts

```diff
@@ -0,0 +1,20 @@
+/********************************************************************************
+ * Copyright (c) 2023 CrossBreeze.
+ ********************************************************************************/
+import { OSUtil, TheiaDialog, USER_KEY_TYPING_DELAY } from '@theia/playwright';
+
+export class TheiaSingleInputDialog extends TheiaDialog {
+    async enterSingleInput(inputValue: string): Promise<void> {
+        const inputField = await this.page.waitForSelector(`${this.blockSelector} .theia-input`);
+        await inputField.press(OSUtil.isMacOS ? 'Meta+a' : 'Control+a');
+        await inputField.type(inputValue, { delay: USER_KEY_TYPING_DELAY });
```

Comment:
`.type` shows up for me as deprecated and we should probably just use `.fill` here. At least for me the test is still green and I don't think we need to have a one-by-one character input here.

Reply:
Good point, I thought the slowed typing was nice, but it is actually slowing down the test :).

I adjusted it.